### PR TITLE
Docs - Change typo in Find command in UG

### PR DESCRIPTION
**Pull request by ruijietay**

### CODE REVIEW

**Review by bgopi23 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/N
 * `RANGE` allows the user to search for a value that falls within the specified `RANGE`. The syntax is as follows:
     * `FROM, TO`. For example, to search for weights that fall between 70kg and 90kg, you can enter `w/70, 90`.
 * Any fields specified in the format above can be searched.
-* When an empty input is specified an optional field, only users that have value(s) available for that field will be showed. (e.g. `find w/` returns all clients that have at least one weight value associated to them).
+* When an empty input is specified for an optional field, only users that have value(s) available for that field will be showed. (e.g. `find w/` returns all clients that have at least one weight value associated to them).
```
Comment (dillontkh) [inline docs/UserGuide.md:273]: Minor grammatical mistake

```suggestion
* When an empty input is specified for an optional field, only users that have value(s) available for that field will be shown. (e.g. `find w/` returns all clients that have at least one weight value associated with them).
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -270,7 +270,7 @@ Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/N
...
 * Multiple fields can be searched in one command.
     * All fields must match (e.g `find n/Wendy p/91234567` will match with a contact whose name **contains** `wendy` and phone number **contains** `91234567`)
```
Comment (bgopi23) [inline docs/UserGuide.md:275]:
```suggestion
    * All fields must match (e.g `find n/Wendy p/91234567` will match with a client whose name **contains** `Wendy` and phone number **contains** `91234567`)
```

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by RyanNgWH — APPROVED** [review]
Looks good!

**Pull request merged**